src/_i18n/es.yml

```diff
@@ -141,9 +141,14 @@ publications-post:
 services:
   title: Software de calidad entregado efectivamente
   subtitle: Lideres reconocidos en continuous delivery y prácticas de calidad y pericia.
+  we-acd: We advise, we coach, we deliver
+  software-craftsmanship-pioneers: Somos pioneros en Software Craftsmanship. Somos conocidos por nuestra experiencia en prácticas XP, Continuous Delivery, Software Architecture, soluciones Cloud e Infraestructura. Aprovechamos nuestra experiencia para ofrecer un servicio a medida de nuestros clientes.
+  we-advise-subtitle: Te asesoramos en la entrega de Software de calidad de forma efectiva.
+  we-coach-subtitle: Te entrenamos en habilidades de diseño de Software y delivery.
```

Comment:
anyway, software -> programas? applicaciones?

Reply:
entrenar in Spanish is only for physical teaching.

Reply:
I'm leaving Software as I don't like the alternatives.
i'm changing `Te entrenamos` for `Te formamos`